### PR TITLE
Improve code readability

### PR DESCRIPTION
**Pull request by ZP-AlwaysWin**
Improve code readability

### CODE REVIEW

File: pkg/qos/queue_sort.go [hunk 1]
```diff
@@ -48,13 +48,16 @@ func (*Sort) Less(pInfo1, pInfo2 *framework.PodInfo) bool {
...
+	case v1.PodQOSGuaranteed:
 		return true
-	}
-	if p1QOS == v1.PodQOSBurstable {
+	case v1.PodQOSBurstable:
 		return p2QOS != v1.PodQOSGuaranteed
+	case v1.PodQOSBestEffort:
+		return p2QOS == v1.PodQOSBestEffort
+	default:
+		panic("Unsupported QOS type")
```
Comment (denkensk) [inline pkg/qos/queue_sort.go:59]: The func `v1qos.GetPodQOS()` won't return other type other than `PodQOSGuaranteed`、`PodQOSBurstable` and `PodQOSBestEffort`.

File: pkg/qos/queue_sort.go [hunk 2]
```diff
@@ -48,13 +48,16 @@ func (*Sort) Less(pInfo1, pInfo2 *framework.PodInfo) bool {
 
 func compQOS(p1, p2 *v1.Pod) bool {
 	p1QOS, p2QOS := v1qos.GetPodQOS(p1), v1qos.GetPodQOS(p2)
-	if p1QOS == v1.PodQOSGuaranteed {
+	switch p1QOS {
```
Comment (Huang-Wei) [inline pkg/qos/queue_sort.go:51]: Hmm, I'm not sure this can improve the readability. To me, it's just stylish difference.

**Pull request closed without merging**